content/logs/languages/php.md

```diff
@@ -0,0 +1,562 @@
+---
+title: PHP log collection
+kind: documentation
+further_reading:
+- link: "/logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "/logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
+- link: "/logs/explore"
+  tag: "Documentation"
+  text: Learn how to explore your logs
+- link: /logs/faq/log-collection-troubleshooting-guide
+  tag: "FAQ"
+  text: Log Collection Troubleshooting Guide
+---
+
+## Overview
+
+Write your PHP logs into a file, then [use the Agent](/logs) to forward them to Datadog, choose between the following libraries:
+
+* [Monolog](/logs/languages/php/#php-monolog)
+* [Symfony](/logs/languages/php/#php-symfony)
+* [Zend-Log](/logs/languages/php/#php-zend-log)
+
+## PHP Monolog
+
+Use Composer to add Monolog as a dependency:  
+ 
+```
+composer require "monolog/monolog"
+```
+
+Alternatively, install it manually:
+
+1. Download Monolog from the repository and include it to the libraries
+2. Initialize the instance in the application's bootstrap
+
+    ```php
+    <?php
+
+    require __DIR__ . '/vendor/autoload.php';
+
+    // load Monolog library
+    use Monolog\Logger;
+    use Monolog\Handler\StreamHandler;
+    use Monolog\Formatter\JsonFormatter;    
+    ```
+
+
+### Setup - Log into a file with Monolog
+
+The following configuration enables the JSON formatting and writes the logs and events into the `application-json.log` file. Edit your code, right after the initialization of the Monolog instance and add a new handler:
+
+```php
+ <?php
+
+require __DIR__ . '/vendor/autoload.php';
+
+// load Monolog library
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\Formatter\JsonFormatter;
+
+// create a log channel
+$log = new Logger('channel_name');
+
+// create a Json formatter
+$formatter = new JsonFormatter();
+
+// create a handler
+$stream = new StreamHandler(__DIR__.'/application-json.log', Logger::DEBUG);
+$stream->setFormatter($formatter);
+
+// bind
+$log->pushHandler($stream);
+
+// an example
+$log->info('Adding a new user', array('username' => 'Seldaek'));
+```
+
+### Configure your Datadog agent
+
+Create a `php.d/conf.yaml` file in your `conf.d/` folder with the following content:
+
+```yaml
+init_config:
+
+instances:
+    
+##Log section
+logs:
+
+    ## - type : file (mandatory) type of log input source (tcp / udp / file)
+    ##   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    ##   service : (mandatory) name of the service owning the log
+    ##   source : (mandatory) attribute that defines which integration is sending the logs
+    ##   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    ##   tags: (optional) add tags to each logs collected
```

Comment:
There's a space between the colon on each of the lines, with the exception of line with `tags:`. Personally, I like the format: `name: ` where there's no space between the name and the colon, but a space following. What do you think?

```yaml
    ## - type: file (mandatory) type of log input source (tcp / udp / file)
    ##   port/path: (mandatory) Set port if type is tcp or udp. Set path if type is file
    ##   service: (mandatory) name of the service owning the log
    ##   source: (mandatory) attribute that defines which integration is sending the logs
    ##   sourcecategory: (optional) Multiple value attribute. Can be used to refine the source attribtue
    ##   tags: (optional) add tags to each logs collected
```